### PR TITLE
Add content to slide file

### PR DESCRIPTION
**Pull request by tpa007**
**TL;DR** A summary of this Pull Request

This pull request:
- Adds content 

Remaining tasks:
- [x] Add another change

cc: @githubteacher

### CODE REVIEW

File: _slides/tpa007.md [hunk 1]
```diff
@@ -3,4 +3,5 @@ layout: slide
 title: Slide for @tpa007
 ---
 
-Replace this line with your favorite meme.
\ No newline at end of file
+At training
```
Comment (githubteacher) [inline _slides/tpa007.md]: 🎉 What do you think about adding more emojis to your slide?

```suggestion
At training 🎉 🎉 🎉 
```

File: _slides/tpa007.md [hunk 2]
```diff
@@ -3,4 +3,5 @@ layout: slide
...
+Enjoying it
```
Comment (githubteacher) [inline _slides/tpa007.md]:
```suggestion
Enjoying it :octocat: :octocat: ❤️ 
```

**Comment by tpa007** [comment]
Thanks for the suggestions! Changes done!

cc: @githubteacher

**Comment by githubteacher** [comment]
#1

**Pull request merged**